dev/Repeater/ViewportManagerWithPlatformFeatures.cpp

```diff
@@ -261,6 +261,7 @@ void ViewportManagerWithPlatformFeatures::OnCacheBuildActionCompleted()
 
 void ViewportManagerWithPlatformFeatures::OnEffectiveViewportChanged(winrt::FrameworkElement const& sender, winrt::EffectiveViewportChangedEventArgs const& args)
 {
+    REPEATER_TRACE_INFO(L"%ls: \tEffectiveViewportChanged event callback \n", GetLayoutId().data());
     UpdateViewport(args.EffectiveViewport());
```

Comment:
Should this be using args.MaxEffectiveViewport()?

Reply:
That is not required for this specific bug. Will be making it as a separate change.